dataform/examples/dataform_datavault/dataform.json

```diff
@@ -0,0 +1,7 @@
+{
+  "warehouse": "bigquery",
+  "defaultSchema": "dataform",
+  "assertionSchema": "dataform_assertions",
+  "defaultDatabase": "YOUR_PROJECT_ID",
```

Comment:
Here I see that users should add their project id. Can you mention this under the steps? If the users have to change more stuff, please add them those as well.